tests/run-make/emit-notify/lib.rs

```diff
@@ -0,0 +1,23 @@
+//@ compile-flags: -C codegen-units=1 --emit asm,llvm-ir,llvm-bc,obj --json=artifacts --error-format=json
```

Comment:
be advised that these compiletest flags **only** affect rmake.rs, not other Rust files, and `compile-flags` don't do anything for `rmake.rs`.

Reply:
Makes sense. It's a leftover from failed attempts to write this test without rmake :)